src/StaticWebAssetsSdk/Targets/Microsoft.NET.Sdk.StaticWebAssets.targets

```
@@ -656,7 +694,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   </Target>
 
-  <Target Name="UpdateExistingPackageStaticWebAssets">
+  <Target Name="UpdateExistingPackageStaticWebAssets" Condition="$(DesignTimeBuild) != 'true'">
```

Nit like everywhere else:

```suggestion
  <Target Name="UpdateExistingPackageStaticWebAssets" Condition="'$(DesignTimeBuild)' != 'true'">
```